dataStores/chat/ChatController.js

```diff
@@ -0,0 +1,32 @@
+const storage = require('../dataStores/Storage');
+
+class ChatController {
+
+	getMessages(req, res) {
+		return res.status(200).json({messages: storage.messages});
+	}
+
+	saveMessage(req, res) {
+		storage.messages.push(req.body.username);
```

Comment:
Czemu do messages zapisujesz username?

Wg danych początkowych chyba powinien to być obiekt podobny do tego {id: uuid(), sender: req.body.username, body: req.body.message, timestamp: Date.now() }, gdzie uuid() to jakiś generator uuidów

Reply:
@J-a-r-e-k @nivlheim prawda